lib/src/command/outdated.dart

```diff
@@ -341,9 +440,20 @@ Future<void> _outputHuman(List<_PackageDetails> rows,
           row.upgradable != row.resolvable)
       .length;
 
+  if (upgradableFailed) {
+    log.message(
+      '\nUpgradable package resolution could not be computed.\n'
```

Comment:
How about: `'\nNo upgrade resolution with no dependency_overrides.\n'

Reply:
from chat, we agreed to drop it.